docs/book/src/user/quick-start.md

```diff
@@ -1091,14 +1091,21 @@ Calico not required for vcluster.
 
 ```bash
 kubectl --kubeconfig=./capi-quickstart.kubeconfig \
-  apply -f https://docs.projectcalico.org/v3.21/manifests/calico.yaml
+  apply -f https://raw.githubusercontent.com/projectcalico/calico/v3.24.1/manifests/calico.yaml
 ```
 
 After a short while, our nodes should be running and in `Ready` state,
 let's check the status using `kubectl get nodes`:
 
 ```bash
 kubectl --kubeconfig=./capi-quickstart.kubeconfig get nodes
+NAME                                          STATUS   ROLES           AGE   VERSION
```

Comment:
I think we followed mostly everywhere to have separate codeblocks for the command and input.

````suggestion
```
```bash
NAME                                          STATUS   ROLES           AGE   VERSION
````

Reply:
Fixed